src/GlobalFooter/GlobalFooter.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { GlobalFooterColors } from './GlobalFooter.levers';
+import { Logo } from '../Logo/Logo';
+import { SrOnlyText } from '../SrOnlyText/SrOnlyText';
+
+/**
+ * Stanford Global Footer Component.
+ *
+ */
+export const GlobalFooter = ({ className, ...props }) => {
+  // Defaults & Variables
+  const classnames = require('classnames');
+  const levers = {};
+
+  // props.color
+  if (props.color && GlobalFooterColors.includes(props.color)) {
+    switch (props.color) {
+      case 'cardinal-red':
+        levers.wrapper = classnames('su-bg-cardinal-red');
+        break;
+
+      case 'digital-red':
+        levers.wrapper = classnames('su-bg-digital-red');
+        break;
+
+      case 'black':
+        levers.wrapper = classnames('su-bg-black');
+        break;
+    }
+  }
+
+  return (
+    <div className={classnames('su-global-footer su-body-basefont-20 su-rs-py-1 su-text-white su-link-white hover:su-link-white focus:su-link-white', levers.wrapper, className)}>
+      <div className='su-cc su-flex su-flex-col lg:su-flex-row' title='Common Stanford resources'>
+        <div className='su-text-center su-mt-5 su-mb-9'>
+          <Logo className={classnames('su-type-3')} color={levers.logo} type="stacked" />
```

Comment:
```suggestion
          <Logo className={classnames('su-type-3')} type="stacked" />
```

levers.logo is never set.

Reply:
Good catch - it's the default color so doesn't need to pass a color.